### PR TITLE
Search bar fix for #234

### PR DESCRIPTION
**Pull request by sachdevs**
#234 Added on window resize handler as a util function instead of directly in `main.js`, hope thats fine.

### CODE REVIEW

File: server/static/js/util.js [hunk 1]
```diff
@@ -323,6 +323,23 @@ function(_, _s, $) {
...
 
   /**
+   * Check window size and remove search bar if too small
+   */
+  var hideSearchIfWindowSmall = function() {
+    var size = $(window).width();
+    // if size of window is less than 1140px, 
+    // the search bar gets push to next line
+    // instead, if size goes to less than 1140, search bar is hidden.
+    if(size < 1140){
```
Comment (mduan) [inline server/static/js/util.js]: nit: space between ) and {

File: server/static/js/util.js [hunk 2]
```diff
@@ -323,6 +323,23 @@ function(_, _s, $) {
...
+      $('.unified-search-bar').hide();
+    }
+    // if search bar was hidden earlier, unhide on window being large enough
+    else if(!$('.unified-search-bar').is(":visible")){
```
Comment (mduan) [inline server/static/js/util.js]: nit: space between ) and {

**Comment by mduan** [comment]
Thanks! The change looks good. I'll test it after work.

**Comment by mduan** [comment]
Sorry, it's gonna take a few more days because my personal laptop was recently formatted. So I have to re-setup my dev environment on it.

**Comment by sachdevs** [comment]
Sure, take your time

File: server/static/js/util.js [hunk 3]
```diff
@@ -323,6 +323,23 @@ function(_, _s, $) {
...
 
   /**
+   * Check window size and remove search bar if too small
+   */
+  var hideSearchIfWindowSmall = function() {
+    var size = $(window).width();
+    // if size of window is less than 1140px, 
+    // the search bar gets push to next line
+    // instead, if size goes to less than 1140, search bar is hidden.
+    if(size < 1140) {
```
Comment (mduan) [inline server/static/js/util.js]: Could you also add a space between `if` and `(`?

File: server/static/js/util.js [hunk 4]
```diff
@@ -323,6 +323,23 @@ function(_, _s, $) {
...
+      $('.unified-search-bar').hide();
+    }
+    // if search bar was hidden earlier, unhide on window being large enough
+    else if(!$('.unified-search-bar').is(":visible")) {
```
Comment (mduan) [inline server/static/js/util.js]: Could you also add a space between `if` and `(`?

**Comment by mduan** [comment]
Checked locally, and the change looks good.

Feel free to add yourself as a contributor in server/templates/about_page.html so that you will show up on https://uwflow.com/about.

**Comment by JGulbronson** [comment]
Hey Saksham, sorry for the long delay. Totally my fault, between Frosh week and my first week of work, I didn't have a ton of free time. Mack says it looks good, so therefore it looks good to me. Can you just squash this down to 1 commit, and then I'll merge?

**Pull request merged**